charts/consul/templates/server-disruptionbudget.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     release: {{ .Release.Name }}
     component: server
 spec:
-  maxUnavailable: {{ template "consul.pdb.maxUnavailable" . }}
+  maxUnavailable: {{ template "consul.server.pdb.maxUnavailable" . }}
```

Comment:
**suggestion:** This is also from #3000 and should be dropped.

Reply:
Corrected as recommended by reverting back to `release/1.3.x` branch version of affected files.

```shell
$ git checkout 'release/1.3.x' -- charts/consul/templates/server-disruptionbudget.yaml charts/consul/test/unit/server-disruptionbudget.bats charts/consul/template/_helpers.tpl
```
Applied datadog-integration changes back into `_helpers.tpl`

Re-ran entirety of bats tests using `Makefile` - `make bats-tests` (all passed)